internal/identity/manager/manager.go

```diff
@@ -288,9 +268,12 @@ func (mgr *Manager) mergeGroups(ctx context.Context, directoryGroups []*director
 				}
 				defer mgr.dataBrokerSemaphore.Release(1)
 
-				_, err = mgr.cfg.Load().dataBrokerClient.Delete(ctx, &databroker.DeleteRequest{
-					Type: any.GetTypeUrl(),
-					Id:   id,
+				_, err = mgr.cfg.Load().dataBrokerClient.Put(ctx, &databroker.PutRequest{
```

Comment:
if there are multiple instances of databroker component running, then there would be concurrent sync processes running as well?

Reply:
yes. They would overwrite each other's changes.

Reply:
i think it deserves a separate discussion https://github.com/pomerium/internal/issues/309